### ISSUE TITLE
Design a system for setting up SheepShaver to run Ares Classic

### ISSUE THREAD

**Issue by gamefreak** [gamefreak/Athena#6]
This can be done with a combination of step by step user instructions and an automated script.
1. Download the SheepShaver "Chubby Bunny" ZIP from [here](http://homepage.mac.com/jlg/COIV4.0.1.zip).
2. Unzip and move the folder to the desired location (Generally /Applications)
3. Select one of the three provided DMGs and unzip it to /Users/Shared
4. Download the Ares installer (possibly fix the type/creator codes)
5. Mount the chosen disk image
6. Copy the Installer to the disk image
7. Unmount the disk image.
8. Launch the SheepShaver app
9. Run the Ares Installer (this may not be scriptable) and install to the blank drive
10. Register Ares (probably unscriptable)

**Comment by TesseractA** [comment]
Is this how you start up Athena? I can't seem to get Athena to run. (On OSX 10.11) @gamefreak or @sfiera

**Comment by sfiera** [comment]
No, this is unrelated. This is just about running Ares in Classic (probably for testing).

Where are you getting Athena from? The build at http://downloads.arescentral.org/Athena/Athena-1.0.zip works for me.

**Comment by TesseractA** [comment]
When I was trying to download Athena, I was trying to download from the page of downloads link 
https://arescentral.org/links/
I could not find the Athena page from the arescentral.org home.
Thank you very much! Do you guys code in C++? I noticed the .ggo file extension.
